pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -1,11 +1,25 @@
 package preflight
 
 import (
-	"errors"
+	//"errors"
 	"fmt"
+	"strconv"
+	"strings"
 
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/oc"
+
+	"github.com/code-ready/crc/pkg/crc/errors"
+	//"github.com/code-ready/crc/pkg/os/windows/win32"
+	"github.com/code-ready/crc/pkg/os/windows/powershell"
+)
+
+const (
+	// Fall Creators update comes with the "Default Switch"
+	minimumWindowsReleaseId = 1709
```

Comment:
@robin-owen This is important to document, as this is not a step we can automate for people. Sure, 1709 was released in 2017 (Sept), but not guaranteed to be installed.

Reply:
Noted, thank you. Once this is merged, we can create a docs issue to get that documented.

Reply:
@robin-owen please go ahead.

but also we need to document that VPN is currently unsupported.